plasmicpkgs/antd5/src/registerSkeleton.tsx

```diff
@@ -0,0 +1,216 @@
+      rootClassName={rootClassName}
+      className={customClassName}
+    />
+  )
+}
+
+export const skeletonComponentName = "plasmic-antd5-skeleton";
+
+export function registerSkeleton(loader?: Registerable) {
+  registerComponentHelper(loader, AntdSkeleton, {
+    name: skeletonComponentName,
+    displayName: 'Skeleton',
+    props: {
+      children: 'slot',
+      type: {
+        type: 'choice',
+        defaultValue: 'Basic',
+        options: ['Basic', 'Button', 'Avatar', 'Input', 'Image', 'Node'],
+      },
+      active: {
+        type: 'boolean',
+        description: 'Show animation effect',
+        defaultValue: false,
+      },
+      avatar: {
+        type: 'boolean',
+        description: 'Show avatar placeholder',
+        hidden: (ps) => ps.type !== 'Basic',
+        defaultValue: false,
+      },
+      loading: {
+        type: 'boolean',
+        description: 'Display the skeleton when true',
+        defaultValue: false,
+      },
+      paragraph: {
+        type: 'boolean',
+        description: 'Show paragraph placeholder',
+        hidden: (ps) => ps.type !== 'Basic',
+        defaultValue: true,
+      },
+      round: {
+        type: 'boolean',
+        description: 'Show paragraph and title radius when true',
+        hidden: (ps) => ps.type !== 'Basic',
+        defaultValue: false,
+      },
+      title: {
+        type: 'boolean',
+        description: 'Show title placeholder',
+        hidden: (ps) => ps.type !== 'Basic',
+        defaultValue: true,
+      },
+      size: {
+        type: 'choice',
+        defaultValue: 'default',
+        description: `Size of the skeleton for types: Avatar, Button and Input`,
+        hidden: (ps) =>
+          ps.type !== 'Avatar' &&
+          ps.type !== 'Button' &&
+          ps.type !== 'Input' &&
+          ps.avatar !== true,
+        advanced: true,
+        options: ['large', 'small', 'default'],
+      },
+      // SkeletonAvatarProps
+      shape: {
+        type: 'choice',
+        defaultValue: 'circle',
+        description: `Set the shape of avatar`,
+        hidden: (ps) => ps.type !== 'Avatar' && ps.avatar !== true,
+        advanced: true,
+        options: ['circle', 'square'],
+      },
+      // SkeletonTitleProps
+      widthTitle: {
+        type: 'string',
+        description: 'Width of the title',
+        hidden: (ps) => !ps.title,
+        advanced: true,
+      },
+      // SkeletonParagraphProps
+      width: {
+        type: 'array',
+        description:
+          'Set the width of paragraph. When width is an Array, it can set the width of each row. Otherwise only set the last row width',
+        hidden: (ps) => !ps.paragraph && ps.type !== 'Basic',
+        advanced: true,
+      },
+      rows: {
+        type: 'number',
+        description: 'Set the row count of paragraph',
+        hidden: (ps) => !ps.paragraph && ps.type !== 'Basic',
+        advanced: true,
+      },
+      // SkeletonButtonProps
+      shapeButton: {
```

Comment:
This does nothing. Also, there's already the `shape` prop that works for this purpose